pkgs/development/tools/quilt/default.nix

```diff
@@ -29,7 +57,8 @@ stdenv.mkDerivation rec {
       and more.
     '';
 
-    license = lib.licenses.gpl2Plus;
+    license = licenses.gpl2Plus;
+    maintainers = with maintainers; [ smancill ];
     platforms = lib.platforms.all;
```

Comment:
Either remove the lib here too or just do `with lib.maintainers; [ ... ]` above.

Reply:
I am blind, I don't know why I didn't see that I also needed to remove `lib` from this line (because I did remove it from the `license` line).

Wasn't `nixpkgs-review` supposed to report this kind of small issues? It didn't show me anything.

Anyway, thanks for the catch, I fixed it.

Reply:
I don't think nixpkgs review does any static analysis. It just builds packages.